worlds/kh1/__init__.py

```diff
@@ -0,0 +1,315 @@
+            for i in range(self.options.accessory_slot_increase):
+                level_up_item_pool.append("Accessory Slot Increase")
+
+            # Create other pool
+            for i in range(self.options.strength_increase):
+                possible_level_up_item_pool.append("Strength Increase")
+            for i in range(self.options.defense_increase):
+                possible_level_up_item_pool.append("Defense Increase")
+            for i in range(self.options.hp_increase):
+                possible_level_up_item_pool.append("Max HP Increase")
+            for i in range(self.options.mp_increase):
+                possible_level_up_item_pool.append("Max MP Increase")
+            for i in range(self.options.ap_increase):
+                possible_level_up_item_pool.append("Max AP Increase")
+
+            # Fill remaining pool with items from other pool
+            while len(level_up_item_pool) < 100 and len(possible_level_up_item_pool) > 0:
+                level_up_item_pool.append(possible_level_up_item_pool.pop(self.random.randrange(len(possible_level_up_item_pool))))
+
+            level_up_locations = list(get_locations_by_category("Levels").keys())
+            self.random.shuffle(level_up_item_pool)
+            i = self.options.force_stats_on_levels - 1
+            while len(level_up_item_pool) > 0 and i < self.options.level_checks:
+                self.multiworld.get_location(level_up_locations[i], self.player).place_locked_item(self.create_item(level_up_item_pool.pop()))
+                i = i + 1
+        
+        #Calculate prefilled locations and items
+        if True: #Allow notepad++ to collpase this section
+            prefilled_items = []
+            prefilled_locations = 1 #Victory
+            if self.options.junk_in_missable_locations:
+                prefilled_locations = prefilled_locations + 15
+            if self.options.vanilla_emblem_pieces:
+                prefilled_locations = prefilled_locations + 4
+                prefilled_items = prefilled_items + ["Emblem Piece (Flame)", "Emblem Piece (Chest)", "Emblem Piece (Fountain)", "Emblem Piece (Statue)"]
+        
+        total_locations = len(self.multiworld.get_unfilled_locations(self.player)) - prefilled_locations
+        
+        non_filler_item_categories = ["Key", "Magic", "Worlds", "Trinities", "Cups", "Summons", "Abilities", "Shared Abilities", "Keyblades", "Accessory", "Weapons", "Puppies"]
+        if self.options.hundred_acre_wood:
+            non_filler_item_categories.append("Torn Pages")
+        for name, data in item_table.items():
+            quantity = data.max_quantity
+            if data.category not in non_filler_item_categories:
+                continue
+            if name in starting_worlds:
+                continue
+            if data.category == "Puppies":
+                if self.options.puppies == "triplets" and "-" in name:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
```

Comment:
```suggestion
                    item_pool += [self.create_item(name) for _ in range(quantity)]
```

For a range, it starts at 0 anyway so you don't need the 0, there